src/main/java/handong/whynot/domain/Post.java

```diff
@@ -40,4 +39,17 @@
 
     @Column(name = "closed_dt")
     public LocalDateTime closedDt;
+
+    @Builder
+    public Post(User createdBy, String title, String content, String postImg) {
+        this.createdBy = createdBy;
+        this.title = title;
+        this.content = content;
+        this.postImg = postImg;
+    }
+
+    public Post addJobs(List<JobPost> jobs) {
```

Comment:
현재 이 메서드를 사용하고 있는 service 코드에서는 리턴되는 Post 객체를 사용하지 않는 것 같습니다. 뭔가 의도나 개인적인 컨벤션이 있는지 궁금합니다. :)

```suggestion
    public void addJobs(List<JobPost> jobs) {
```

Reply:
네네 현재는 사용하고 있지 않은데요. 
네네 그 메서드를 검증하기 위한 용도로 사용하려고 일부러 해둔 것은 맞습니다!

Reply:
이 부분은 조금 고민해봐야할것같아요. 메서드를 실제로 사용하는 비즈니스로직에서 void 메서드처럼사용하고 있는데 실제로는 반환값이 있는 건... 🤔 
이 메서드를 검증하기 위해서 꼭 반환값이 있어야할지 생각해보면 좋을 것 같습니다.